src/backend/XmlHttpRequest.ts

```diff
@@ -69,11 +75,12 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
     if (typeof(opts.index) === "string") {
       XmlHttpRequest.FromURL(opts.index, cb, opts.baseUrl, false);
     } else {
-      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false));
+      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use));
```

Comment:
So technically, I think adding try/catch to .FromURL would *also* prevent any unhandled errors in .Create... but I'm going to add try/catch to .Create as well just so we don't shoot ourselves in the foot later if we add another kind of thrown error to the constructor.